src/test/projects/hello-world-with-test/pom.xml

```diff
@@ -0,0 +1,23 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>fr.inria.gforge.spoon</groupId>
+  <artifactId>hello-world</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>fr.inria.gforge.spoon</groupId>
+        <artifactId>spoon-maven-plugin</artifactId>
+        <configuration>
```

Comment:
the processor should be configured in full classpath

Reply:
I dont understand what you mean sry. Could you explain it?

Reply:
> the processor should be configured in full classpath

@monperrus  BTW could you explain why you've removed fullclasspath by default? It seemed like it would have better chance to work by default than having to force a configuration parameter, no? What am I missing? And yes I'm sure there are good reasons I don't know, I haven't followed this discussion/decision. I'm just curious to know the reasons. Thx!

Reply:
> I dont understand what you mean sry. Could you explain it?

I think @monperrus means using `<noclasspath>false</noclasspath>` or something like this but I'm not sure either.

Reply:
> @monperrus BTW could you explain why you've removed fullclasspath by default?

It was removed from spoon-core by default, we did not make any explicit decision for spoon-maven-plugin, it inherits the default of spoon-core.

Why we set the default to NOCLASSPATH in spoon-core? Because it mawimizes the chance that a newcomer gets something interesting out of spoon without reading the doc and without setting a correct full classpath.